index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <!-- Nav Bar menu on mobile -->
+  <header>
+    <nav class="mobile-nav-bar">
+      <h1>My Logo</h1>
+      <img src="Assets/Unionhamb.png" alt="mobile menu">
+    </nav>
+  </header>
+
+  <!-- Section Introduction and -->
+
+  <section class="intro-section">
+    <h1 class="intro-section__title">I’m Arthur. 
+    Glad to see you!</h1>
+    <p class="intro-section__subtitle">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <p class="intro-section__capstext">LET’S CONNECT</p>
```

Comment:
For the sake of semantic correction and the screen readers kindly use `hx (examples: h2, h3, h4, h5, h6)` instead of the `p` tag.